apps/spruce/src/pages/image/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "@emotion/styled";
+import { H2, Overline } from "@leafygreen-ui/typography";
+import { ImageTabRoutes } from "constants/routes";
+import { size } from "constants/tokens";
+import { getTabTitle } from "./getTabTitle";
+
+interface HeaderProps {
+  imageId: string;
+  tab: ImageTabRoutes;
+}
+
+export const Header: React.FC<HeaderProps> = ({ imageId, tab }) => {
+  const { title } = getTabTitle(tab);
+
+  return (
+    <Container data-cy="image-tab-title">
+      <Overline>{title}</Overline>
```

Comment:
Using `Overline` the way it's actually meant to be used 🫡 

That being said, we usually don't use the "entity" being viewed as the heading on these pages. Is there a particular reason to deviate from that pattern here? i.e. I would expect the heading to say "Build Information" and the image in question is present in the sidebar selector

Reply:
It comes from the feedback during the design phases, where users were having trouble orienting themselves in the app. They couldn't tell what image they were looking at (even with the name written in the dropdown).

From the feedback:
<img width="412" alt="Screenshot 2024-08-13 at 1 43 37 PM" src="https://github.com/user-attachments/assets/2c81aa0c-f92c-4df4-ba4d-6106c7fe88ca">

I think maybe we should rework the project one too tbh, there's been some history of people not noticing they're on the wrong project and making edits to it. Example in [DEVPROD-3137](https://jira.mongodb.org/browse/DEVPROD-3137)

Reply:
Great context, thanks!